plugins/Bookmark.jsx

```diff
@@ -121,6 +130,19 @@ class Bookmark extends React.Component {
             location.href = url;
         }
     };
+    zoomToBookmarkExtent = (bookmarkkey) => {
+        resolveBookmark(bookmarkkey, (params) => {
+            if ('c' in params && 's' in params) {
+                const scale = parseFloat(params.s);
+                const zoom = MapUtils.computeZoom(this.props.mapScales, scale);
+                const center = params.c.split(/[;,]/g).map(x => parseFloat(x));
+                this.props.zoomToPoint(center, zoom, this.props.mapCrs);
+            } else if ('e' in params) {
+                const bounds = (params.e).split(',').map(n => parseFloat(n));
+                this.props.zoomToExtent(bounds, this.props.mapCrs);
```

Comment:
Sorry I forgot earlier, there is also the `urlPositionCrs` config option, which will add a `crs` URL parameter if `urlPositionCrs` is different than `mapCrs`. Something like `params.crs ?? this.props.mapCrs` should do.

Reply:
Thanks, I was not aware of these options